docs/content/guides/installation/install/index.md

```diff
@@ -0,0 +1,255 @@
+  docker push ${TARGET_REGISTRY}/${IMAGE}:${VERSION}
+done
+```
+Then install Radius configured to pull images from your private registry, and supply image pull secrets if authentication is required.
+
+```bash
+rad install kubernetes \
+  --set global.imageRegistry=myregistry.azurecr.io \
+  --set global.imageTag={{ .Chart.AppVersion }} \
+  --set global.imagePullSecrets[0].name=myregistry-secret
+```
+
+When using a custom registry, images are pulled directly from <registry>/<image-name>:<tag> format. For example, with myregistry.azurecr.io, the controller image will be pulled from myregistry.azurecr.io/controller:latest.
+
+#### Configure workload identity
+
+Radius enables you to deploy and connect to cloud resources across Azure and AWS. The chart flags `global.azureWorkloadIdentity.enabled` and `global.aws.irsa.enabled` toggle the Kubernetes-side configuration to use workload identity; you still need to configure cloud identities and register credentials afterward. See the [Azure workload identity guide]({{< ref "/guides/operations/providers/azure-provider/howto-azure-provider-wi" >}}) and the [AWS IRSA guide]({{< ref "/guides/operations/providers/aws-provider/howto-aws-provider-irsa" >}}).
+
+```bash
+# Azure workload identity
+rad install kubernetes --set global.azureWorkloadIdentity.enabled=true
+
+# AWS IAM Roles for Service Accounts
+rad install kubernetes --set global.aws.irsa.enabled=true
+```
+
+#### Pre-download Terraform binaries
+
+Radius can pre-stage Terraform binaries inside the control-plane pods so Terraform-based Recipes start faster and do not need internet access at runtime. Enable it and optionally point to a custom download URL:
+
+```bash
+rad install kubernetes --set global.terraform.enabled=true
+# Optional custom artifact mirror
+rad install kubernetes --set global.terraform.enabled=true --set global.terraform.downloadUrl=https://example.com/terraform_1.6.6_linux_amd64.zip
+```
+
+#### Configure observability endpoints
+
+Control-plane components expose Prometheus metrics and Zipkin-compatible tracing. Override the defaults to point at your platform endpoints during installation:
+
+```bash
+# Custom metrics scrape path and TLS proxy port
+rad install kubernetes --set global.prometheus.path=/metrics-radius --set global.prometheus.port=9443
+
+# Send traces to Jaeger/Zipkin collector
+rad install kubernetes --set global.zipkin.url=http://jaeger-collector.radius-monitoring.svc.cluster.local:9411/api/v2/spans
+```
+
+Refer to the [Prometheus]({{< ref "/guides/operations/control-plane/metrics/prometheus" >}}) and [Jaeger]({{< ref "/guides/operations/control-plane/traces/jaeger" >}}) guides for end-to-end configuration steps.
+
+#### Run pre-upgrade checks automatically
+
+Enable the `preupgrade` job to validate Helm connectivity, version compatibility, and cluster prerequisites before each upgrade:
+
+```bash
+rad install kubernetes --set preupgrade.enabled=true --set preupgrade.checks.resources=true
```

Comment:
These properties (preupgrade.enabled and preupgrade.checks.*) are for `rad upgrade` not for `rad install`